File: pigatize.rb
def starts_with_vowel?(text)
  /[aeiou]/i === text[0]
end

def pigatize(text)
  if starts_with_vowel?(text)
    puts "#{text}way"
  else
    puts "#{text[1..-1]}#{text[0]}ay"
  end
end

pigatize("stop")
pigatize("go")
pigatize("apple")

File: pigatize.js
starts_with_vowel = function (text) {
  return /[aeiou]/gi.test(text[0]);
};

pigatize = function (text) {
  if (starts_with_vowel(text)) {
    console.log(text+"way");
  } else {
    console.log(text.substring(1,text.length)+text[0]+"ay");
  }
};

pigatize("apple");
pigatize("thing");
